docs/usage.md

```diff
@@ -44,20 +45,74 @@ TREATMENT_REP3,AEG588A6_S6_L003_R1_001.fastq.gz,
-| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `sample`  | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
-| `fastq_1` | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
-| `fastq_2` | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+| Column         | Description                                                                                                                                                                            |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `sample`       | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
+| `fastq_1`      | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+| `fastq_2`      | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
+## Assembler options
+
+By default, the pipeline uses Megahit (i.e. `--assembler  megahit`) to assemble the cleaned and trimmed FastQ reads to create the reference genome. Megahit is fast and it requires a not a lot of memory to run, typically is suggested to be used with prokaryotic samples. The pipeline allows you to choose another assembler RNAspades, (i.e. `--assembler rnaspades` ), that is usually suggested to use for eukaryotic samples.
+
+> N.B. you can use `Megahit` for eukaryotic samples too, we just suggest what is the best option according to our experience (literature?).
+
+## Orf caller options
+
+By default, the pipeline uses prodigal (i.e. `--orf_caller prodigal` ) to generate the genome feature file (.gff) and to generate gene structure from the assembly.
+
+Other orf caller options for running the pipeline are:
+
+- Prokka (`--orf_caller prokka`)
+
+- Transdecoder (`--orf_caller transdecoder`)
+
+> N.B. Prokka and prodigal are suggested to run with prokaryotes while transdecoder is specific for eukaryotes.
```

Comment:
This is just a potential change, probably because I like "recommended" more than "suggested", if you prefer suggested, then just keep it:
```suggestion
> N.B. Prokka and prodigal are recommended for prokaryotic datasets while transdecoder is specific for eukaryotic datasets.
```